duckduckgo/Android
# Remove previous Location screen

Task/Issue URL: 

### Description
Remove code related to previous Location screen and edit style names related to permissions dialogs

### Steps to test this PR

_Optional_
- Install from this branch
- Go to Settings > Site Permissions
- [ ] Check all works as expected

### No UI changes
